backend/danswer/llm/utils.py

```diff
@@ -244,6 +245,25 @@ def message_generator_to_string_generator(
         yield message_to_string(message)
 
 
+def message_to_message_chunk(message: BaseMessage) -> MessageChunk:
+    if not isinstance(message.content, str):
+        raise RuntimeError("LLM message not in expected format.")
+
+    keyword_arguments = message.additional_kwargs
+
+    return MessageChunk(
+        content=str(message.content),
+        tokens=keyword_arguments["usage_metadata"]["output_tokens"],
```

Comment:
nit: should this gracefully handle the case where this is missing?

Reply:
Definitely- should be accounted for in c5dcc49564adc642d9ac251b67e5aca253582192